src/expressions.rst

```diff
@@ -3419,7 +3422,7 @@ Closure Expressions
 
 :dp:`fls_2d141c9a0yui`
 A :t:`closure expression` is an :t:`expression` that defines a :t:`closure
-type`.
+type` and constructs a value of such a :t:`type`.
```

Comment:
The following might sounds a bit better:

A closure expression is an expression that defines a closure type and constructs a value of that type.